src/main/graphql-server/README.md

```diff
@@ -0,0 +1,29 @@
+# GraphQL Tools
+
+## Capture
+
```

Comment:
Is a capture a request associated with some snapshot of code? Might be worth a sentence or two to explain

Reply:
I wouldn't say a snapshot of code. More a snapshot of behavior. This is useful to ensure the behavior is as it was at the time of capture.